experimental/examples/bubblebench/editable-shared-tree/README.md

```diff
@@ -0,0 +1,35 @@
+# @fluid-example/bubblebench-editable-shared-tree
+
+## Getting Started
+
+You can run this example using the following steps:
+
+1. Run `npm install` and `npm run build:fast -- --nolint` from the `FluidFramework` root directory.
+   a. For an even faster build, you can add the package name to the build command, like this:
+   `npm run build:fast -- --nolint @fluid-example/bubblebench`
+1. Run `npm run start` from this directory (experimental/examples/bubblebench/sharedtree) and open <http://localhost:8080> in a web browser to see the app running.
+ <!-- AUTO-GENERATED-CONTENT:END -->
```

Comment:
This section is out of date (does not reflect the correct package name). You need to rerun the markdown generator thing that generates these sections. See "markdown-magic" in `docs/README.md`.

Reply:
Done in latest commit 4d265